here is error from hack/verify-golint.sh:
```
+ for err in '"${errors[@]}"'
+ echo 'pkg/volume/util/resize_util.go:30:24: should omit type map[v1.PersistentVolumeClaimConditionType]bool from declaration of var knownResizeConditions; it will be inferred from the right-hand side'
pkg/volume/util/resize_util.go:30:24: should omit type map[v1.PersistentVolumeClaimConditionType]bool from declaration of var knownResizeConditions; it will be inferred from the right-hand side
+ echo

+ echo 'Please review the above warnings. You can test via "golint" and commit the result.'
Please review the above warnings. You can test via "golint" and commit the result.
+ echo 'If the above warnings do not make sense, you can exempt this package from golint'
If the above warnings do not make sense, you can exempt this package from golint
+ echo 'checking by adding it to hack/.golint_failures (if your reviewer is okay with it).'
checking by adding it to hack/.golint_failures (if your reviewer is okay with it).
+ echo

+ false
++ kube::log::errexit
++ local err=1
++ set +o
++ grep -qe '-o errexit'
++ set +o xtrace
!!! Error in hack/verify-golint.sh:114
  Error in hack/verify-golint.sh:114. 'false' exited with status 1
Call stack:
  1: hack/verify-golint.sh:114 main(...)
Exiting with status 1
```

i am looking into the errs from test

--- a/cmd/kubeadm/app/util/arguments.go
+++ b/cmd/kubeadm/app/util/arguments.go
@@ -30,26 +30,26 @@ import (
 func BuildArgumentListFromMap(baseArguments map[string]string, overrideArguments map[string]string) []string {
 	var command []string
 	var keys []string
-	for k := range overrideArguments {
+
+	argsMap := make(map[string]string)
+
+	for k, v := range baseArguments {
+		argsMap[k] = v
+	}
+
+	for k, v := range overrideArguments {
+		argsMap[k] = v
+	}
+
+	for k := range argsMap {
 		keys = append(keys, k)
 	}
+
 	sort.Strings(keys)
 	for _, k := range keys {
-		v := overrideArguments[k]
-		// values of "" are allowed as well
-		command = append(command, fmt.Sprintf("--%s=%s", k, v))
+		command = append(command, fmt.Sprintf("--%s=%s", k, argsMap[k]))
 	}
-	keys = []string{}
-	for k := range baseArguments {
-		keys = append(keys, k)
-	}
-	sort.Strings(keys)
-	for _, k := range keys {
-		v := baseArguments[k]
-		if _, overrideExists := overrideArguments[k]; !overrideExists {
-			command = append(command, fmt.Sprintf("--%s=%s", k, v))
-		}
-	}
+
 	return command
 }
 

--- a/cmd/kubeadm/app/util/arguments_test.go
+++ b/cmd/kubeadm/app/util/arguments_test.go
@@ -85,9 +85,9 @@ func TestBuildArgumentListFromMap(t *testing.T) {
 			},
 			expected: []string{
 				"--admission-control=NamespaceLifecycle,LimitRanger",
-				"--something-that-allows-empty-string=",
 				"--allow-privileged=true",
 				"--insecure-bind-address=127.0.0.1",
+				"--something-that-allows-empty-string=",
 			},
 		},
 	}

